webui/react/src/utils/loadable.ts

```diff
@@ -94,6 +114,12 @@ type MatchArgs<T, U> =
  */
 const match = <T, U>(l: Loadable<T>, cases: MatchArgs<T, U>): U => {
   switch (l._tag) {
+    case 'Errored':
+      return 'Errored' in cases
+        ? cases.Errored(l.error)
+        : 'NotLoaded' in cases
+        ? cases.NotLoaded()
+        : cases._();
```

Comment:
I made this part flexible so `Errored` case is optional but we may want to make this required for all loadable states?